src/applications/PerfusionAlignment.cpp

```diff
@@ -16,4 +16,34 @@ See COPYING file or https://www.med.upenn.edu/cbica/captk/license.html
 
 #include "PerfusionAlignment.h"
 
-
+#include <vtkChartXY.h>
+#include <vtkTable.h>
+#include <vtkFloatArray.h>
+#include <vtkPlot.h>
+
+void PerfusionAlignment::Createchart(vtkChartXY* chart, std::map<std::string, std::vector<float>*>* map)
```

Comment:
```suggestion
void PerfusionAlignment::Createchart(vtkChartXY* chart, std::map<std::string, std::vector<float>> map)
```

Reply:
No change needed. What we have is correct and avoids duplication of memory.